### ISSUE TITLE
failure install into Laravel: using composer require oseintow/laravel-bigcommerce

### ISSUE THREAD

**Issue by redbonzai** [oseintow/laravel-bigcommerce#2]
The response that is returned is : 
[InvalidArgumentException]                                                                                                                                    
  Could not find package oseintow/laravel-bigcommerce at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability

This is strange because I checked Packagist.org, and it is definitely there. 

How do I fix this issue ? 
If it helps, this is a fresh Laravel install, as I am beginning a fresh API project. 

Thank you for your time. 
Christian Dominguez

**Comment by oseintow** [comment]
check your composer.json. It could be the minimal-stability is set to stable and you are installing the dev-master branch. You can require the 1.0.0 version or changed minimal-stability to dev.

**Comment by redbonzai** [comment]
Thank you, I did check that out.  The problem is that I do have PHP7 installed on  OS X, but composer thinks I am using PHP 5.6.  This is also despite the fact that I checked the path on .bash_profile ( it also says php 7). 
Anyway, I restarted the system, and re exported the path, and then just added your package under the require section inside composer.json.  
After doing a composer update your module was included within the project.  
Thank you so much for your response. 
Christian
